discord/ext/tasks/__init__.py

```diff
@@ -752,15 +743,6 @@ def loop(
     """
 
     def decorator(func: LF) -> Loop[LF]:
-        return Loop[LF](
-            func,
-            seconds=seconds,
-            minutes=minutes,
-            hours=hours,
-            count=count,
-            time=time,
-            reconnect=reconnect,
-            loop=loop,
-        )
+        return Loop[LF](func, seconds=seconds, minutes=minutes, hours=hours, count=count, time=time, reconnect=reconnect)
```

Comment:
What's the point of this change?

Reply:
The `loop` attribute was removed from this class